schemas/com.snowplowanalytics.snowplow.ecommerce/promotion/jsonschema/1-0-0

```diff
@@ -0,0 +1,62 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a promotion entity in Ecommerce",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.ecommerce",
+        "name": "promotion",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "id": {
+            "description" : "The ID of the promotion.",
+            "type": "string",
+            "maxLength": 4096
+        },
+        "name": {
+            "description" : "The name of the promotion.",
+            "type": [ "string", "null" ],
+            "maxLength": 256
+        },
+        "products" : {
```

Comment:
Nit: why not call this `product_ids` directly? We have the `id` field in the `product` schema, so I think naming it `product_ids` would make the relation more explicit.